src/baldr/geojson.cc

```diff
@@ -19,14 +19,13 @@ template <class coord_t>
 MapPtr to_geojson(const typename midgard::GriddedData<coord_t>::contours_t& grid_contours, bool polygons, const std::vector<std::string>& colors) {
   //for each contour interval
   int i = 0;
-  auto color_itr = colors.cbegin();
+  auto color_itr = colors.crbegin();
```

Comment:
this doesnt seem right to me. it was broken before but this doesnt fix it.. i think

so the `grid_contours` are sorted from "largest" to "smallest", right? we need to take care to sort the `colors` vector accordingly. below (and in the service) it looks to me like we just use the `colors` in the order that they appear in the input which is to say there is no sorting enforced. what we need to do is sort the `colors` in the same order that `grid_contours` are sorted, ie by the time value.